src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,54 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    protected void getStatistic(String fromFileName, String toFileName) {
+        String[] fileInfo = this.readFromFile(fromFileName);
+        int supplySum = 0;
+        int buySum = 0;
+        for (String s : fileInfo) {
+            String[] stringInfo = s.split(",");
+            if (stringInfo[0].equals("supply")) {
```

Comment:
Please extract strings/other stuff in constants if they used more than once or represents magic numbers

Reply:
Make constants for all magic number
VALUE_INDEX = 1, OPERATION_TYPE_INDEX = 0